src/idsk/components/intro-block/intro-block.scss

```diff
@@ -211,11 +223,8 @@
     }
 
     .app-pane-grey {
-        .govuk-heading-s,
-        .idsk-intro-block__side-menu__a,
-        .idsk-intro-block__side-menu__li {
+        .govuk-heading-s {
             color: govuk-colour("black");
-            font-weight: bold;
         }
         .idsk-intro-block__side-menu__text {
             margin-top: govuk-spacing(3);
```

Comment:
app-pane-grey

Reply:
Túto časť môžem prakticky úplne vymazať nie? :D Kedže ten komponent to aký je teraz nijako. nedovoluje, aby existoval element s tou classou